spec.yaml

```diff
@@ -1039,6 +1039,109 @@ paths:
         "401":
           $ref: "./responses.yaml#/responses/V4Generic401Response"
 
+  /v4/clusters/{cluster_id}/status/:
+    get:
+      operationId: getClusterStatus
+      tags:
+        - clusters
+      summary: Get cluster status
+      description: |
+        Returns an object about a cluster's current state and past status transitions.
+
+        This endpoint passes through the status content of the Kubernetes resources representing
+        a cluster in the according custom resource. That is, depending on the provider:
+
+        - [awsconfig.provider.giantswarm.io](https://godoc.org/github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1#AWSConfig)
```

Comment:
We might even want to link directly to the Status resource here: 

https://godoc.org/github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1#AWSConfigStatus

Reply:
I didn't do it now, because I think it's better to point to the CR we mention. The users will find their way there though.